cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -195,10 +195,18 @@ def load(
     ) -> Properties:
         props: MutableMapping[ViewId, MutableMapping[PropertyIdentifier, PropertyValue]] = {}
         for space, view_properties in data.items():
+            if not isinstance(view_properties, dict):
```

Comment:
Why would this ever not be a dict?

Reply:
I realize that this might be overkill, but if any thing else than an object is returned here then the current code would break without this extra check.
![image](https://github.com/cognitedata/cognite-sdk-python/assets/60234212/d797ec2b-95f0-4bc6-95b5-9035baedae08)

Reply:
That won't happen, don't need to take that into consideration.